protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -1147,7 +1174,7 @@ func renderServices(services []*descriptor.Service, paths openapiPathsObject, re
 					if parameter.IsRepeated() {
 						core := schemaCore{Type: paramType, Format: paramFormat}
 						if parameter.IsEnum() {
-							var s []string
+							var s interface{}
 							core.Enum = enumNames
 							enumNames = s
```

Comment:
I'm confused by this code, but I think we can just do this assignment directly instead
```suggestion
							enumNames = nil
```

Reply:
Nice catch, let's fix this.